package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "restaurant-page",
+  "version": "1.0.0",
+  "description": "Simple restaurant page",
+  "main": "webpack-config.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node app",
+    "build": "webpack",
+    "watch": "npx webpack --watch"
+  },
```

Comment:
KIndly replace the script with `## Status: Require changes :recycle: :traffic_light: 

Hi @300ms, great job on this project :+1: However, there a few comments we need to address before we can proceed, kindly check the comments accordingly :smile: 
- Kindly create a detailed `README` file for this project
- Let us replace our script with `"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
`

Reply:
Hey ^^
I've added and edit current "scripts" just like you asked. I didn't have webpack-dev-server installed in my project but now I do.
But the command is not "webpack-dev-server --mode development --open".
It is "npx webpack serve" since it has been changed on the webpack side. 
(Please see the: https://github.com/webpack/webpack-dev-server/issues/2759#issuecomment-706675162)